backend/tests/Squidex.Domain.Users.Tests/Squidex.Domain.Users.Tests.csproj

```diff
@@ -15,9 +15,12 @@
   <ItemGroup>
     <PackageReference Include="FakeItEasy" Version="5.4.0" />
     <PackageReference Include="FluentAssertions" Version="5.9.0" />
+    <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="1.1.0" />
```

Comment:
Whats that?

Reply:
Reverted. There were issues while building "Squidex.Web.Tests", "Squidex.Infrastructure.Tests" and "Squidex.Domain.Users.Tests" projects, so I had to add these dependencies, but now it works without them as well. Probably I had to delete the NuGetFallbackFolder after installing 3.0 SDK, that might have caused this issue.